ansible/site.yml

```diff
@@ -31,10 +30,106 @@
   become: True
   roles:
     - common
+
+- name: Install keystone
+  hosts: controllers
+  remote_user: root
+  vars_files:
```

Comment:
@Click2Cloud-Gamma Hi,Gamma,I think just keep the var_files that roles need to use, and other unused var_files should be removed.

Reply:
I have removed unused group_vars

Reply:
Hi @lijuncloud  I have made all the changes you have suggested. Thanks!